Engines/Wine/Verbs/VK9/script.js

```diff
@@ -101,15 +101,12 @@ class VK9 {
     static install(container) {
         const wine = new Wine();
         const wizard = SetupWizard(InstallationType.VERBS, "VK9", Optional.empty());
-
-        wine.prefix(container);
+        
+        wine.prefix(container); 
+        const versions = getGithubReleases("disks86", "VK9", wizard);
+        const selectedVersion = wizard.menu(tr("Please select the version."), versions, versions[0]);
         wine.wizard(wizard);
```

Comment:
Can you please group these statements so that they are semantically more correct:
1) initialize wine:
   ```
   wine.wizard(wizard);
   wine.prefix(container);
   ```
2) afterwards select the verb version:
   ```
   const versions = ...
   const selectedVersion = ....
   ```

This order makes it easier to follow the code flow and it also allows for an easier extension of the verb.

Reply:
A separator line between these two segments is also useful :)

Reply:
Done